index.js

```diff
@@ -0,0 +1,76 @@
+const uuid = require('node-uuid');
+
+/**
+ * Filter for response messages
+ * @param  {object}  msg - message to check
+ * @return {Boolean}
+ */
+function isChildMessage(msg) {
```

Comment:
The main reason is that we require a bind on `isChildMessage` though have no nice way of documenting that.